x-pack/plugins/security_solution/public/users/pages/navigation/user_risk_tab_body.tsx

```diff
@@ -0,0 +1,130 @@
+import { useQueryToggle } from '../../../common/containers/query_toggle';
+import { UserRiskScoreQueryId, useUserRiskScore } from '../../../risk_score/containers';
+import { buildUserNamesFilter } from '../../../../common/search_strategy';
+import { UsersComponentsQueryProps } from './types';
+import { UserRiskInformationButtonEmpty } from '../../components/user_risk_information';
+
+const QUERY_ID = UserRiskScoreQueryId.USER_DETAILS_RISK_SCORE;
+
+const StyledEuiFlexGroup = styled(EuiFlexGroup)`
+  margin-top: ${({ theme }) => theme.eui.paddingSizes.l};
+`;
+
+const UserRiskTabBodyComponent: React.FC<
+  Pick<UsersComponentsQueryProps, 'startDate' | 'endDate' | 'setQuery' | 'deleteQuery'> & {
+    userName: string;
+  }
+> = ({ userName, startDate, endDate, setQuery, deleteQuery }) => {
+  const timerange = useMemo(
+    () => ({
+      from: startDate,
+      to: endDate,
+    }),
+    [startDate, endDate]
+  );
+
+  const { toggleStatus: overTimeToggleStatus, setToggleStatus: setOverTimeToggleStatus } =
+    useQueryToggle(`${QUERY_ID} overTime`);
+  const { toggleStatus: contributorsToggleStatus, setToggleStatus: setContributorsToggleStatus } =
+    useQueryToggle(`${QUERY_ID} contributors`);
+
+  const [loading, { data, refetch, inspect }] = useUserRiskScore({
+    filterQuery: userName ? buildUserNamesFilter([userName]) : undefined,
+    onlyLatest: false,
+    skip: !overTimeToggleStatus && !contributorsToggleStatus,
+    timerange,
+  });
+
+  useQueryInspector({
+    queryId: QUERY_ID,
+    loading,
+    refetch,
+    setQuery,
+    deleteQuery,
+    inspect,
+  });
+
+  const toggleContributorsQuery = useCallback(
+    (status: boolean) => {
+      setContributorsToggleStatus(status);
+    },
+    [setContributorsToggleStatus]
+  );
+
+  const toggleOverTimeQuery = useCallback(
+    (status: boolean) => {
+      setOverTimeToggleStatus(status);
+    },
+    [setOverTimeToggleStatus]
+  );
+
+  const rules = data && data.length > 0 ? data[data.length - 1].risk_stats.rule_risks : [];
+
+  return (
+    <>
+      <EuiFlexGroup direction="row">
+        <EuiFlexItem grow={2}>
+          <RiskScoreOverTime
+            from={startDate}
+            to={endDate}
+            loading={loading}
+            riskScore={data}
+            queryId={QUERY_ID}
+            title={i18n.USER_RISK_SCORE_OVER_TIME}
+            toggleStatus={overTimeToggleStatus}
+            toggleQuery={toggleOverTimeQuery}
+          />
+        </EuiFlexItem>
+
+        <EuiFlexItem grow={1}>
+          <TopRiskScoreContributors
+            loading={loading}
+            queryId={QUERY_ID}
+            toggleStatus={contributorsToggleStatus}
+            toggleQuery={toggleContributorsQuery}
+            rules={rules}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <StyledEuiFlexGroup gutterSize="s">
+        {/* // TODO PENDING ON USER RISK DOCUMENTATION
```

Comment:
I left 3 TODOs in the original issue to make sure I don't forget to remove these comments. 
https://github.com/elastic/kibana/issues/129046